modules/settings/variables.tf

```
@@ -13,6 +13,15 @@
 # ------------------------------------------------------
 # TFE
 # ------------------------------------------------------
+variable "daemonauthpassword" {
+  default     = null
+  type        = string
+  description = <<-EOD
+  (Optional) This Password allows access to the daemon authentication - may be set if going between installations.
+  If unset, it will default to the random .
+  EOD
+}
```

```suggestion
variable "daemon_auth_password" {
  default     = null
  type        = string
  description = <<-EOD
  (Optional) This password allows access to the daemon authentication. If unset, it will create a new one.
  EOD
}
```

Just a little tidying up of the wording. 

Also, can you move this to the `Replicated` section in alphabetical order, so after `bypass_preflight_checks`?